src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoroelasticKernel.hpp

```diff
@@ -293,9 +274,9 @@ class SinglePhase :
 
     for( integer a = 0; a < numNodesPerElem; ++a )
     {
-      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][0] * m_biotCoefficient * detJxW;
-      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][1] * m_biotCoefficient * detJxW;
-      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][2] * m_biotCoefficient * detJxW;
+      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][0] * dPorosity_dVolStrainIncrement * detJxW;
+      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][1] * dPorosity_dVolStrainIncrement * detJxW;
+      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][2] * dPorosity_dVolStrainIncrement * detJxW;
```

Comment:
Here we need the following derivative: `dTotalStress_dPressure`.

Reply:
true, but aren't the two derivatives the same? I thought they were and didn't want to create and extra variable but correct me if I am wrong.